mobile/fastlane/metadata/android/fr-FR/full_description.txt

```diff
@@ -0,0 +1,10 @@
+Cozy Cloud permet à chacun d’avoir son cloud personnel, respectueux de la vie privée. Avec Cozy Files, accédez aux fichiers et aux photos que vous stockez dans votre Cozy de partout.
```

Comment:
Lets go for "smartphone/tablette" instead of "ordiphone" we're not from Quebec :)